src/main/java/seedu/chatteroo/ui/Ui.java

```diff
@@ -0,0 +1,57 @@
+package seedu.chatteroo.ui;
+
+import seedu.chatteroo.tasks.Task;
+
+/**
+ * Represents the user interface of the Chatteroo ChatBot program.
+ */
+public class Ui {
+    /**
+     * Constructor for the Ui class.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Shows the welcome text.
+     */
+    public static String showWelcomeText() {
+        return "Hello! I'm Chatteroo!\n" + "What can I do for you?";
+    }
+
+    /**
+     * Shows the exit text.
+     */
+    public String showByeText() {
+        return "Chatteroo says bye and hopes to see you again soon!\n";
+    }
+
+    /**
+     * Shows the specified text when the list contains no tasks.
+     */
+    public void showNoTaskText() {
+        System.out.println("There are no tasks in the list.\n");
+    }
+
+    public String showAddTaskText(Task newTask, int listCount) {
```

Comment:
Perhaps instead of naming them as showAddTaskTest you could say that it is a response to the user and name it as
showAddTaskResponse? Similarly for the rest as well.